src/gui/widgets/unit_preview_pane.cpp

```diff
@@ -145,6 +145,13 @@ static inline std::string get_hp_tooltip(const utils::string_map& res, const std
 
 static inline std::string get_mp_tooltip(int total_movement, std::function<int (t_translation::terrain_code)> get)
 {
+	auto movement_compare = [](std::pair<t_string, int> a, std::pair<t_string, int> b)
+	{
+		return a.first < b.first;
```

Comment:
I just noticed this is actually wrong. You shouldn't use `operator<` with `t_string` values, it won't give the correct result. Instead use either `translation::compare` or `translation::icompare` (depending on whether you want a case-sensitive comparison).

Reply:
You mean changing it to look like this?
```
auto movement_compare = [](std::pair<t_string, int> a, std::pair<t_string, int> b)
{
		return translation::icompare(a.first, b.first);
};
```
This is not producing correct results for English, anyway. Same with ``translation::compare``.

Reply:
`icompare` and `compare` have weird C-like interfaces similar to `std::string::compare`. You need `translation::compare(a, b) < 0;`.